src/components/Loading.js

```diff
@@ -0,0 +1,25 @@
+import { h } from 'preact'
+import { useSoftkey, useI18n } from 'hooks'
+export const Loading = ({ close, message }) => {
+  const i18n = useI18n()
+
+  const onClose = () => {
+    if (close) {
```

Comment:
Is the `close` property used anywhere?

Reply:
I'm thinking of `<Loading/>` can be used either as a single page or popup content, but yes we are only using this in article so far, so I will remove this line until we need it.